sim-lib/src/lnd.rs

```diff
@@ -88,6 +81,28 @@ impl LightningNode for LndNode {
         &self.info
     }
 
+    async fn get_network(&mut self) -> Result<Network, LightningError> {
+        let info = self
+            .client
+            .lightning()
+            .get_info(GetInfoRequest {})
+            .await
+            .map_err(|err| LightningError::GetInfoError(err.to_string()))?
+            .into_inner();
+
+        if info.chains.is_empty() {
+            return Err(LightningError::GetInfoError(
+                "LND node is not connected to a single chain".to_string(),
+            ));
+        }
+
+        Ok(Network::from_str(match info.chains[0].network.as_str() {
+            "mainnet" => "bitcoin",
+            x => x,
```

Comment:
"simnet" is making this unhappy: 
```
Error: Lightning Error: GetInfoError("failed to parse simnet as network")

Caused by:
    Get info error: failed to parse simnet as network
```

Reply:
Of course LND has its own regtest 🤦 

I'll patch that one too.